index.html

```diff
@@ -248,5 +248,86 @@ <h3>Skills</h3>
+                     text ever since the 1500s, when an unknown printer took a galley 
+                     of type and scrambled it 1960s with the releaLorem Ipsum is simply 
+                     dummy text of the printing and typesetting industry. Lorem Ipsum has 
+                     been the industry's standard dummy text ever since the 1500s, when an 
+                     unknown printer took a galley of type and scrambled it 
+                    1960s with the relea</p></div>
+        </div>
+    </section>
+    <section class="background-pop">
+        <div class="pop-mobile-container">
+            <div class="img-pop-holder">
+                <img src="Mobile-pop.png">
+                <div class="pop-mobile-close">X</div>
+                </div>
+                <h2 class="mobile-pop-title">Keeping track of hundreds of
+                    components</h2>
+                    <ul class="pop-prog-flex">
+                        <li class="prog-li-pop">Ruby on rails</li>
+                        <li class="prog-li-pop">css</li>
+                        <li class="prog-li-pop">Javascript</li>
+                       </ul>
+                       <div class="mobile-msg-pop">
+                       <p>
+                        Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                     Lorem Ipsum has been the industry's standard dummy 
+                     text ever since the 1500s, when an unknown printer took a galley 
+                     of type and scrambled it 1960s with the releaLorem Ipsum is simply 
+                     dummy text of the printing and typesetting industry. Lorem Ipsum has 
+                     been the industry's standard dummy text ever since the 1500s, when an 
+                     unknown printer took a galley of type and scrambled it 
+                    1960s with the relea
+                       </p>
+                    </div>
+                    <div class="button-p btn-pop"><button class="trans-btn pop-btn">See live<img class="pop-icon" src="Iconseelive.png"></button><button class="trans-btn pop-btn">See source<img class="pop-icon" src="Vectorgithub.png"></button></div>
+        </div>
+    </section>
+    <script>
+        const seeproject = document.querySelector('.popup-d');
+const seebtn = document.querySelector('#myBtn');
+const closebtn = document.querySelector('.close-background');
+
+seebtn.onclick = function aaa() {
+  seeproject.style.display = 'block';
+};
+
+closebtn.onclick = function bbb() {
+  seeproject.style.display = 'none';
+};
+
+     var seeprojectm = document.querySelector(".pop-mobile-container");
+var seebtnm = document.querySelector("#myBtn");
+var closebtnm = document.querySelector(".pop-mobile-close");
+
+seebtnm.onclick = function() {
+  seeprojectm.style.display = "block"
+};
+
+closebtnm.onclick = function() {
+  seeprojectm.style.display = "none"
+};
+
+        </script>
```

Comment:
- [ ] Good job adding functionality to your project :+1:, but kindly remember that as a project requirement, the JavaScript code should be implemented following the [best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md), it means to create its own file and not use an `<script>` tag. I would make your project looks professional :+1: